### DISCUSSION TITLE
[request] Add option to remove non-selected playlist items

### DISCUSSION THREAD

**Discussion by fulalas** [audacious-media-player/audacious#77] [Ideas]
In Winamp we have this option 'crop selected' that is very handy (although the name is not exactly intuitive):

![1](https://github.com/audacious-media-player/audacious/assets/27843666/38f80d48-b739-461b-939b-391d051ccacb)

Result:
![2](https://github.com/audacious-media-player/audacious/assets/27843666/2493a280-bfba-48b1-993d-84e14aba3421)

Basically, it removes all items of the playlist that are not selected. And the shortcut is `ctrl+delete`.

Thanks for the hard work! Audacious rocks! :)

**Comment by jlindgren90** [comment]
Note that you can do this in Audacious with Invert Selection + Remove Selected.

**Comment by fulalas** [comment]
Yeah, I know. But it's not as convinient as the `ctrl+delete` feature, don't you think?

**Reply by jlindgren90** [reply]
It seems a fairly uncommon operation. In over a decade, I don't recall ever hearing it requested before. Convenience is nice but we also don't want to clutter the UI too much. Saving one user one click is not worth confusing 100 other users.

**Comment by wildstar84** [comment]
Hey @fulalas - Curious, what skin are you using?  It looks nice, sort of reminds me a bit of my Solaris/CDE days!

**Reply by jlindgren90** [reply]
That's Winamp with the default look (at least in old versions).
